### PR TITLE
Update Documentation on how to update post meta values from a block.

### PR DESCRIPTION
**Pull request by jorgefilipecosta**
## Description
The meta attributes were deprecated, but our docs were not updated.
This PR updates the tutorial on how to create a meta block to use the new mechanisms.

### CODE REVIEW

File: docs/designers-developers/developers/tutorials/metabox/meta-block-3-add.md [hunk 1]
```diff
@@ -4,9 +4,7 @@ With the meta field registered in the previous step, next you will create a new
 
 For this block, you will use the TextControl component, which is similar to an HTML input text field. For additional components, check out the [components](/packages/components/src) and [editor](/packages/editor/src/components) packages repositories.
 
-Attributes are the information displayed in blocks. As shown in the block tutorial, the source of attributes come from the text or HTML a user writes in the editor. For your meta block, the attribute will come from the post meta field.
-
-By specifying the source of the attributes as `meta`, the block editor automatically handles the loading and storing of the data; no REST API or data functions are needed.
+The hook `useEntityProp` can be used by the blocks to change meta values.
```
Comment (ajitbohra) [inline docs/designers-developers/developers/tutorials/metabox/meta-block-3-add.md]:
```suggestion
The hook `useEntityProp` can be used by the blocks to get or change meta values.
```

Comment (jorgefilipecosta) [reply docs/designers-developers/developers/tutorials/metabox/meta-block-3-add.md]: Thank you for the review @ajitbohra 👍

**Comment by mgburns** [comment]
@jorgefilipecosta Just a heads up that I attempted to follow the page sourced from this guide yesterday with WP 5.3.2 and it failed due to `useEntityProp` being undefined:
https://developer.wordpress.org/block-editor/tutorials/metabox/meta-block-3-add/

Is it normal workflow to update the dev docs with Gutenberg features that haven't landed in core yet?

**Comment by jorgefilipecosta** [comment]
Hi @mgburns, yes, we usually update the documentation in the plugin even for changes that are not yet part of the last release (e.g., things that are only available in the plugin).
This process is far from perfect, and we need some way to specify which version the documentation refers too. I'm sorry for the trouble caused.

**Comment by alex-418** [comment]
Hi, just wanted to note the developer handbook still uses the deprecated meta attributes instead of this new mechanism: https://developer.wordpress.org/block-editor/tutorials/metabox/meta-block-3-add/

**Comment by TimothyBJacobs** [comment]
@jorgefilipecosta Yeah, this is confusing that meta attributes are still recommended here as well: https://developer.wordpress.org/block-editor/reference-guides/block-api/block-attributes/#meta

**Comment by jorgefilipecosta** [comment]
> Hi, just wanted to note the developer handbook still uses the deprecated meta attributes instead of this new mechanism: [developer.wordpress.org/block-editor/tutorials/metabox/meta-block-3-add](https://developer.wordpress.org/block-editor/tutorials/metabox/meta-block-3-add/)

> @jorgefilipecosta Yeah, this is confusing that meta attributes are still recommended here as well: [developer.wordpress.org/block-editor/reference-guides/block-api/block-attributes/#meta](https://developer.wordpress.org/block-editor/reference-guides/block-api/block-attributes/#meta)



It seems documentation for both sections uses the new way of doing this or refers the meta attribute source as deprecated. Let me know if further updates are required.

**Pull request merged**